### ISSUE TITLE
Ethos 1.5.2 Can't select File Path in text display widget X18

### ISSUE THREAD

**Issue by pmullen503** [FrSkyRC/ETHOS-Feedback-Community#3640]
X18
1.5.2 firmware

I'm trying to add a text widget to the display on my X18.  I can create the text widget but can't select the file path. I see"---" but trying to select just exits back to the widget page.

I have a :documents/user folder and .txt file in both the Radio and SD card drives.

**Comment by bsongis-frsky** [comment]
Would you paste your TXT file.
It should be available either on your Nand or your SD depending on the System settings, in directory /documents/user/

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/7c8836cf-dfc6-41b9-b05e-ec277875c01a)

**Comment by strgaltdel** [comment]
maybe a typo in your folder names?

it should work
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/20392683/ce2cce98-01bb-46de-9526-d99a32d6041c)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/20392683/56b0f27c-ffb2-49b3-b619-5b1eb539573b)


![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/20392683/e2b8b931-09e5-409e-a58a-7bef1a969efb)



.. sorry, parallel posting

**Comment by bsongis-frsky** [comment]
Screenshots are obviously easier to understand :)

**Comment by pmullen503** [comment]
Here's where I started:
![screenshot-2024-03-12-75733](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/11808346/2f63ca09-8a46-4c74-a3be-d60c8026d219)
![screenshot-2024-03-12-75941](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/11808346/6af1bf98-4018-4ab0-9c2b-0c7b4f3498b8)

After changing the storage location to RADIO It started working:

![screenshot-2024-03-12-78770](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/11808346/9fe50e24-4c86-4200-8321-e21fa3a38090)

I changed the storage location back to the SD card and it also started working there (the documents directory was copied onto both).

All I can deduce from that is after update the file location had been corrupted or otherwise not set and resetting it fixed it.

Bottom line is it's apparently working now.
